cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1 on 2024-12-15 13:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(null=True, upload_to=""),
```

Comment:
Consider specifying a directory for the `upload_to` parameter in the `ImageField`. This will help organize the uploaded images in a specific folder within your media directory.